server/src/main/java/org/room/apollo/server/util/HttpUtil.java

```diff
@@ -0,0 +1,57 @@
+package org.room.apollo.server.util;
+
+import org.springframework.stereotype.Component;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+@Component
+public class HttpUtil {
```

Comment:
Do we need to use HttpURLConnection? Is it better to remove this class and use ResTemplate?

Reply:
RestTemplate

Reply:
Looks good, will change